src/androidTest/java/com/couchbase/lite/DatabaseEncryptionTest.java

```diff
@@ -230,4 +234,67 @@ public boolean update(UnsavedRevision rev) {
         Assert.assertNotNull(seekrit);
         Assert.assertEquals(1, seekrit.getDocumentCount());
     }
+
+    public void testEncryptedAttachments() throws Exception {
+        if (!isSQLiteDB())
+            return;
+
+        Manager cryptoManager = getEncryptionTestManager();
```

Comment:
I intended to give it the named so that we know that we are getting a manager with an encryption. Nothing more than this.